packages/stripe-graphql/src/index.ts

```diff
@@ -0,0 +1,142 @@
+import * as dotenv from 'dotenv'
+import { GraphQLScalarType } from 'graphql'
+import Stripe from 'stripe'
+
+import { CORSOptions, createServer, GraphQLYogaError, YogaInitialContext } from '@graphql-yoga/node'
+
+dotenv.config()
```

Comment:
not sure we should add `dotenv` to the package - we could let the user decide to use it
or then, to make it global to the `functions` environment?